### ISSUE TITLE
About total_loss = (a_loss + c_losss).mean()

### ISSUE THREAD

**Issue by KGBUSH** [MorvanZhou/pytorch-A3C#18]
I suppose in the paper, authors maybe calculate the loss for actor and critics respectively. I was wondering it really works better when you do "total_loss = (a_loss + c_losss).mean()" in the loss_func()? 

Thanks

**Comment by jakkarn** [comment]
The sum of gradients is the same as gradients of the sum.
